### DISCUSSION TITLE
Display reverse field where the item is "used in"

### DISCUSSION THREAD

**Discussion by rokdd** [directus/directus#16862] [Archive]
I have the problem that sometimes the context of the item is not very clear for the user. So I want to create a panel or read-only field where the user can see, which other items of another collection are refering to this item. Is there a good example how to achieve this? Do I need to adjust a panel or create a new field?

**Comment by rijkvanzanten** [comment]
Sounds like you're looking for the one-to-many interface 🙂

**Reply by rokdd** [reply]
Right but which displays a existing relations as read-only? How can use the interface without creating a new field?

**Comment by burka** [comment]
@rokdd did you find any solution for this? We are looking into the same problem :-)

**Reply by rokdd** [reply]
@burka Well I am trying to make it work.. it breaks somehow sometimes and I try to figure out why. I followed basically this thread: https://github.com/directus/directus/discussions/9350
